scripts/verification.py

```diff
@@ -59,10 +77,10 @@
 
 nl_state = newline.is_there(file)
 
-if nl_state:
+if nl_state is 1:
```

Comment:
Please use the `==` operator for integer comparison. I've seen it in various places and I'm not going to mark them all, since a simple sed invocation should do the trick.

Reply:
Done that in c941bd625c38b1bbdd523e39ac262cf6867ced69